tests/plugins/toolsets/bash/test_argv_security.py

```
@@ -0,0 +1,315 @@
+    allow list, review that command's dangerous arguments (see
+    _dangerous_argv_reason) and add coverage, THEN update the expected set below.
+    """
+
+    EXPECTED_CORE = {
+        "kubectl get", "kubectl describe", "kubectl logs", "kubectl top",
+        "kubectl explain", "kubectl api-resources", "kubectl config view",
+        "kubectl config current-context", "kubectl cluster-info", "kubectl version",
+        "kubectl auth can-i", "kubectl diff", "kubectl events",
+        "jq", "grep", "head", "tail", "sort", "uniq", "wc", "cut", "tr",
+        "id", "whoami", "hostname", "uname", "date", "which", "type", "echo",
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)